app/assets/javascripts/autocomplete.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function() {
```

Comment:
I think changing it now is best, just so we can set the right example. I'll switch to React :)